packages/nimble-components/src/rich-text/components/mention-view/mention-users-view/styles.ts

```diff
@@ -0,0 +1,38 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+    mentionFont,
+    mentionFontColor,
+    mentionDisabledFontColor
+} from '../../../../theme-provider/design-tokens';
+import { iconAtTag } from '../../../../icons/at';
+
+export const styles = css`
+    ${display('inline-block')}
+
+    :host {
+        box-sizing: border-box;
+        font: ${mentionFont};
+        white-space: normal;
+        --ni-nimble-icon-color: ${mentionFontColor};
+    }
+
+    .control {
+        color: ${mentionFontColor};
```

Comment:
Can just move this style under `:host`

Reply:
As per the [mockup](https://www.figma.com/file/JYi8RTEDGARU3YJf22zXkH/SystemLink-LIMS---BTS?node-id=7248%3A114263&mode=dev), we just want the green font only after a selection. So can this be styled as-is?
![image](https://github.com/ni/nimble/assets/123377523/6618db6b-abae-4eb6-9e60-fcf8204f268e)

Reply:
I think that intent would be more clearly expressed as:
```
:host {
   color: bodyFontColor or whatever it is
}

.control {
   color: mentionFontColor
}
```